test/controllers/contentProperty.controller.test.js

```diff
@@ -548,4 +549,147 @@ describe('Content Property Controller', () => {
       });
     });
   });
+
+  describe('Get all content properties', () => {
+    const dummyProperties = ContentPropertyFactory.buildList(18, { areaId });
+
+    beforeEach(async () => {
+      await ContentProperty.insertMany(dummyProperties);
+    });
+    const defaultPaginationResult = {
+      currentPage: 1,
+      limit: 10,
+      offset: 0,
+      result: 10,
+      total: 18,
+      totalPage: 2,
+    };
+
+    describe('when properties exist in db', () => {
+      context('when no parameters are passed', () => {
+        it('returns the default values', (done) => {
+          request()
+            .get('/api/v1/content-property/all')
+            .set('authorization', editorToken)
+            .end((err, res) => {
+              expectsPaginationToReturnTheRightValues(res, defaultPaginationResult);
+              done();
+            });
+        });
+      });
+
+      context('when editor token is used', () => {
+        it('returns the given page and limit', (done) => {
+          request()
+            .get('/api/v1/content-property/all?page=2&limit=5')
```

Comment:
Since you have tested this above, you can use the one without page and limit to make the test simpler:

```suggestion
            .get('/api/v1/content-property/all')
```